content/security/committers.md

```diff
@@ -137,12 +147,12 @@ employer) at the discretion of the project's security team, providing that
 you make clear that the information is not for public disclosure and that you copy to
 `security@apache.org` or the project's security mailing list any communication regarding the vulnerability.
 
-# CVE names
+<h2 id="ids">CVE IDs<a class="headerlink" href="#ids" title="Permanent link">&para;</a></h2>
```

Comment:
I wonder if this section is meant to mean CVE IDs, or if it's meant to be about CPEs. It feels a bit redundant a section if all it's talking about is IDs, the processes above explain that only the Security Team get IDs.

Reply:
I think you're roughly right. Searching around, this page talks about a "CVE name": https://manuals.gfi.com/en/languard/content/acm/topics/appendix/common_vulnerabilities_and_exposures__cve_.htm

It appears that a CVE name is a larger identifier that includes a CVE ID as well as its status, description etc. Which is a really weird bit of naming. I think colloquially we would just call that a CVE, and I think having this section explain CVE IDs works well enough without it needing to get into the innards of CVEs.

Resolving this conversation :)